prow/gerrit/client/client.go

```diff
@@ -468,6 +468,7 @@ func (h *gerritInstanceHandler) queryAllChanges(lastState map[string]time.Time,
 			}).Error("Failed to query changes")
 			continue
 		}
+		log.WithFields(logrus.Fields{"Project": project, "Changes": len(changes)}).Infof("Finished querying for changes.")
```

Comment:
I don't think this function is used any more, probably should have been deleted. https://github.com/kubernetes/test-infra/blob/82e04845eb3ffccb067d2fafcbdc43f62003e1f0/prow/gerrit/adapter/adapter.go#L306 is probably where this change should be